packages/core/src/driver.ts

```diff
@@ -288,6 +382,106 @@ class Driver {
     this._afterConstruction()
   }
 
+  /**
+   * The bookmark managed used by {@link Driver.executeQuery}
+   *
+   * @experimental
+   * @type {BookmarkManager}
+   * @returns {BookmarkManager}
+   */
+  get queryBookmarkManager (): BookmarkManager {
+    return this._queryBookmarkManager
+  }
+
+  /**
+   * Executes a query in a retriable context and returns a {@link EagerResult}.
+   *
+   * This method is a shortcut for a transaction function.
```

Comment:
maybe document that `CALL {} IN TRANSACTIONS` and the older `... USING PERIODIC COMMIT` queries won't work here.